validation/management/commands/writetranscriptions.py

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env python3
+# -*- coding: UTF-8 -*-
+
+# Copyright 2021 Jolene Poulin <jcpoulin@ualberta.ca>
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from pathlib import Path
+import sqlite3
+import os
+import shutil
+from contextlib import closing
+
+from django.core.management.base import BaseCommand, CommandError  # type: ignore
+
+
+class Command(BaseCommand):
+    """
+    Takes all extracted audio files in .wav format
+    and writes transcription files for them that can
+    then be used by Persephone and Simple4All.
+    Files are created in the ./audio directory and stored by speaker
+    """
+
+    help = "creates transcription files for Persephone and Simple4All"
+
+    def handle(self, *args, **options):
+        # Change this if the raw audio files exist elsewhere!
+        _path = "./audio"
+
+        # Generate this folder by running:
+        # python3 manage.py importrecordings --wav --skip-db
+        audio_dir = Path(_path)
+        for audio_file in os.listdir(audio_dir):
+            audio_id = audio_file[:-4]
+
+            with closing(sqlite3.connect("./db.sqlite3")) as conn:
+                cur = conn.cursor()
+
+                query = """
+                select speaker_id, transcription from validation_recording as rec, validation_phrase as p
+                where rec.id = ?
+                and rec.phrase_id = p.id;
+                """
+
+                cur.execute(query, (str(audio_id),))
+                result = cur.fetchone()
+                speaker = result[0] if result else None
+                transcription = result[1] if result else None
```

Comment:
If performance becomes an issue, you could perform a [Split Loop refactoring](https://refactoring.com/catalog/splitLoop.html), and open the database one, and fetch ALL of the mappings from {audio_id: (speaker_id, transcription)} from the database in one go.

Reply:
Every time you look at this chunk of code, you find a way to make it more efficient 😂

Reply:
I mean, don't optimize until you know it's a problem, but I figure opening and closing the database over 100,000 times is the kind of thing to that ends up becoming a performance problem, haha!